sql-parser/src/main/antlr/SqlBase.g4

```diff
@@ -329,7 +330,6 @@ setCmpQuantifier
 datetimeLiteral
```

Comment:
we can probably drop the `datetimeLiteral` rule?

Reply:
yeah, I guess we could drop the whole thing?

@mfussenegger any reason to keep this?

Reply:
No, if it doesn't break anything?